metrics/report/report_dockerfile/dut-details.R

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env Rscript
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+suppressMessages(suppressWarnings(library(tidyr))) # for gather()
+library(tibble)
+library(gridExtra)	# So we can plot multiple graphs together
+suppressMessages(suppressWarnings(library(ggpubr)))		# for ggtexttable
+suppressMessages(library(jsonlite))		# to load the data
+
+resultsfiles=c(
+	"boot-times.json",
+	"footprint-busybox-ksm.json",
+	"footprint-elasticsearch-ksm.json",
+	"footprint-mysql-ksm.json",
+	"memory-footprint.json",
+	"memory-footprint-ksm.json"
+	)
+
+data=c()
+stats=c()
+stats_names=c()
+
+# For each set of results
+for (currentdir in resultdirs) {
+	count=1
+	dirstats=c()
+	for (resultsfile in resultsfiles) {
+		fname=paste(inputdir, currentdir, resultsfile, sep="/")
+		if ( !file.exists(fname)) {
+			#warning(paste("Skipping non-existent file: ", fname))
+			next
+		}
+
+		# Derive the name from the test result dirname
+		datasetname=basename(currentdir)
+
+		# Import the data
+		fdata=fromJSON(fname)
+
+		if (length(fdata$'kata-env') != 0 ) {
+			# We have kata-runtime data
+			dirstats=tibble("Run Ver"=as.character(fdata$'kata-env'$Runtime$Version$Semver))
+			dirstats=cbind(dirstats, "Run SHA"=as.character(fdata$'kata-env'$Runtime$Version$Commit))
```

Comment:
$This$makes$my$eyes$bleed$but$I$guess$you$cannot$change$it

Reply:
hah - nope, can't really change it. That is the format you get when you use [jsonlite](https://cran.r-project.org/web/packages/jsonlite/index.html) to read JSON files into an R data format. Yeah, it is 'deep', and you get the funny quote mark magic to work around non-var-type characters in the names. 
The only way to make this slightly simpler is probably to extract a copy of the `kata-env` subtree, so we could drop a couple of `$` bits from the front of the subsequent references. Probably not worth it in this case.

Oh, for any R aware readers out there - I meant to apologise for my R code - I'm not an R expert, and am learning whilst going along... ;-)